frontend/src/components/Admin/AdminShoutouts/AdminShoutouts.tsx

```diff
@@ -173,7 +175,18 @@ const AdminShoutouts: React.FC = () => {
 
   useEffect(() => {
     ShoutoutsAPI.getAllShoutouts().then((shoutouts) => setAllShoutouts(shoutouts));
-  }, []);
+
+    const shoutoutCollection = collection(db, 'shoutouts');
+    const unsubscribe = onSnapshot(shoutoutCollection, (snapshot) => {
+      const newShoutouts = snapshot.docs.map((docSnapshot) => {
+        const data = docSnapshot.data() as Shoutout;
+        return data;
+      });
+
+      setAllShoutouts(newShoutouts);
+    });
+    return unsubscribe;
+  }, [setAllShoutouts]);
```

Comment:
I'm not sure if we want to setup the snapshot listener for admin shoutouts for all view modes (i.e. not just presentation mode). Might be something that @andrew032011 and @JacksonStaniec can weigh in on.